### PR TITLE
uses file path not url when creating configmap

### PR DESCRIPTION
**Pull request by dixudx**
Wrong change involved by #7398.

Flag `--from-file` uses file path, not urls.

Fixes kubernetes/kubernetes#64007

### CODE REVIEW

**Comment by steveperry-53** [comment]
/hold
This requires a discussion in SIG Docs about how we handle config files throughout the entire doc set. Let's wait until we decide on an overall policy.

**Comment by Freeaqingme** [comment]
@steveperry-53 is it perhaps possible, for now, to embed (part of) that file on that page, or link to the github url where it currently resides (if my understanding is correct)? I'm asking, cause as a k8s novice I got kind of stuck on this page, and can imagine others having some difficulties with it as well.

Also, as part of the getting started tasks, this is so far the only time I've encountered a 404 error, so even if the problem pertains to the entire doc set, perhaps a - albeit temporary - quick fix for this particular page can be thought of?

**Comment by steveperry-53** [comment]
@Freeaqingme,
Yes, a temporary fix would be fine.

**Comment by zacharysarah** [comment]
@dixudx Bumping for action on reviews.

**Comment by dixudx** [comment]
@zacharysarah @steveperry-53 @tengqm Done. PTAL.

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 1]
```diff
@@ -44,13 +44,16 @@ You can use `kubectl create configmap` to create a ConfigMap from multiple files
 For example:
 
 ```shell
-kubectl create configmap game-config --from-file=https://k8s.io/docs/tasks/configure-pod-container/configmap/kubectl
+mkdir -p configure-pod-container/configmap/kubectl/
+wget https://k8s.io/configure-pod-container/configmap/kubectl/game.properties -o configure-pod-container/configmap/kubectl/game.properties
```
Comment (tengqm) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md]: The url here is obviously wrong!

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 2]
```diff
@@ -44,13 +44,16 @@ You can use `kubectl create configmap` to create a ConfigMap from multiple files
...
+wget https://k8s.io/configure-pod-container/configmap/kubectl/ui.properties -o configure-pod-container/configmap/kubectl/ui.properties
```
Comment (tengqm) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md]: here too

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 3]
```diff
@@ -154,8 +157,8 @@ Use the option `--from-env-file` to create a ConfigMap from an env-file, for exa
 #   Blank lines are ignored.
 #   There is no special handling of quotation marks (i.e. they will be part of the ConfigMap value)).
 
-
-cat docs/tasks/configure-pod-container/game-env-file.properties
+wget https://k8s.io/configure-pod-container/configmap/kubectl/game-env-file.properties -o configure-pod-container/configmap/kubectl/game-env-file.properties
```
Comment (tengqm) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md]: here too

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 4]
```diff
@@ -193,9 +196,10 @@ metadata:
 When passing `--from-env-file` multiple times to create a ConfigMap from multiple data sources, only the last env-file is used:
 
 ```shell
+wget https://k8s.io/configure-pod-container/configmap/kubectl/ui-env-file.properties -o configure-pod-container/configmap/kubectl/ui-env-file.properties
```
Comment (tengqm) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md]: here too

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md]: Also don't get the idea why we use such a complicated local path for experiments.

**Comment by zacharysarah** [comment]
@tengqm If you're satisfied with the edits, please `/lgtm` and `/hold cancel`

/approve
/hold

**Pull request merged**